infrastructure/zk/package.json

```diff
@@ -32,6 +32,6 @@
         "@types/tabtab": "^3.0.1",
         "hardhat": "=2.16.0",
         "typescript": "^4.3.5",
-        "cspell": "^8.2.3"
+        "cspell": "*"
```

Comment:
i'd prefer to specify the version

Reply:
The reason I dont is that we are using at `latest` in the dockerfile so to keep consistency between CI environment and what it is reporting and local development.

Reply:
ok